tsrc/cli/env_setter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Dict
+
+from tsrc.git import Status as GitStatus
+from tsrc.repo import Repo
+from tsrc.workspace import Workspace
+
+
+class EnvSetter:
+    def __init__(self, workspace: Workspace):
+        self.workspace = workspace
+
+    def get_shared_env(self) -> Dict[str, str]:
+        return get_workspace_vars(self.workspace)
+
+    def get_env_for_repo(self, repo: Repo) -> Dict[str, str]:
```

Comment:
Nitpick: this class is very empty, I would have made this a free function.

Reply:
Yeah, but get_env_for_repo needs a workspace and combines the other free functions, so ...

I'll remove get_shared_env which is indeed useless.